samples/hello_world/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,3 @@
+&uart0 {
```

Comment:
Does this belong here?  It doesn't look like it belongs here....

Reply:
Of course not. I grabbed this change accidentally. Shame on me.